### PR TITLE
Readme update and install enable/disable feature 

### PR DESCRIPTION
**Pull request by schnoog**
Added a lock-file (support/allow_install) to enable or disable (deleting the file) the installer
Create/delete  also possible on settings.php
README updated (removed old stuff)
install.php removed hint options for manually support/dba.php creation since support directory required write permissons. Is also check during install.
Language files updated

New bash script language_extract.sh
- requires xgettext (which is available for unix / linux / windows
- searches for gettext calls in all php files (outside of smartyfolders & vendors)
- if text not yet in templates/php_language_extract.tpl it will be added
-- REMARK: 	It isn't able to extract escaped text so gettext("Hello \"You\"") will be skipped
--			Use gettest('Hello "You"') instead and everything is fine (hopefully)
- afterwards the languages/storage.pot will be updated

### CODE REVIEW

File: languages/locale/en_GB/messages.po [hunk 1]
```diff
@@ -868,6 +868,80 @@ msgstr "Access to the following directories was denied:"
...
+msgid "Die Installation ist momentan nicht erlaubt"
+msgstr "Installation is currently not allowed"
+
+#: templates//php_language_extract.tpl:86
+msgid "Einträge wurden gelöscht."
+msgstr "Entries were deleted."
+
+#: templates//php_language_extract.tpl:87
+#, fuzzy
+#| msgid "Einzelwert: Zeichenfolge mit bis zu < 512 Zeichen"
```
Comment (jrie) [inline languages/locale/en_GB/messages.po:897]: Should be 256 characters, not 512.

File: languages/locale/en_GB/messages.po [hunk 2]
```diff
@@ -868,6 +868,80 @@ msgstr "Access to the following directories was denied:"
...
+msgid "Einzelwert: Zeichenfolge mit bis zu < 256 Zeichen"
+msgstr "Single value: String with less than 512 character"
```
Comment (jrie) [inline languages/locale/en_GB/messages.po:899]: Should be 256 characters, not 512.

File: languages/sqstorage.pot [hunk 3]
```diff
@@ -822,6 +822,57 @@ msgstr ""
...
+msgid "E-Mail-Adresse ungültig!"
+msgstr ""
+
+#: templates//php_language_extract.tpl:90
+msgid ""
+"Erstelle eine Datei mit dem Namen allow_install im Verzeichnis support um "
+"die Installation zu erlauben."
+msgstr ""
+
+#: templates//php_language_extract.tpl:91
```
Comment (jrie) [inline languages/sqstorage.pot:859]: Should `php_language_extract.tpl` really be considered for translation?

**Comment by schnoog** [comment]
php_language_extract.tpl is only a storage for more easy storage.pot creation.
The tsmarty2c.php script mentioned in the LANGUAGE.md only crawls only the templates for extraction of gettext strings to update the storage.pot
Therefore the language_extract.sh searches in the php files and extracts the strings used there. 

Without this an update of the storage.pot from templates may drop all msgids only within the php files.

**Pull request merged**